jetty-core/jetty-server/src/test/java/org/eclipse/jetty/server/ErrorProcessorTest.java

```diff
@@ -71,21 +73,21 @@ public void before() throws Exception
             @Override
             public void process(Request request, Response response, Callback callback)
             {
-                if (request.getPathInContext().startsWith("/badmessage/"))
+                if (Request.getPathInContext(request).startsWith("/badmessage/"))
```

Comment:
Store `Request.getPathInContext(request)` in a local variable to avoid calling it multiple times.